### PR TITLE
Add 3 adhoc rscripts

### PR DESCRIPTION
**Pull request by pbchase**
I need to share these before someone else needs them. Some might yet get turned into something more formal.

### CODE REVIEW

File: adhoc_analysis_of_contact_info.R [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  mutate(quarantine_date_passed = case_when(
+    contact_quarantine_date < now() ~ TRUE,
+    TRUE ~FALSE
+  ))
+
+# %>%
+#   select(record_id, redcap_event_name, contact_method_prefer, contact_method, contact_released_from_quarantine, cleared_for_campus)
+
+set_contact_method_prefer %>%
+  group_by(contact_method) %>%
```
Comment (ljwoodley) [inline adhoc_analysis_of_contact_info.R:29]:
```suggestion
  group_by(contact_method_prefer) %>%
```

File: adhoc_check_for_missing__cleared_for_campus_timestamp__when__cleared_for_campus__is_set.R [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# If you get any records from these three queries, figure out how it happened then fix the data
+case_records %>%
+  select(record_id, contains("cleared_for_campus")) %>%
+  filter(is.na(cleared_for_campus) != is.na(cleared_for_campus_timestamp))
+
+contact_records %>%
+  select(record_id, contains("cleared_for_campus")) %>%
+  filter(is.na(cleared_for_campus) != is.na(cleared_for_campus_timestamp))
+
+result_records %>%
```
Comment (ljwoodley) [inline adhoc_check_for_missing__cleared_for_campus_timestamp__when__cleared_for_campus__is_set.R:44]: This script doesn't read https://redcap-phone.ctsi.ufl.edu/prod/redcap_v10.0.1/index.php?pid=31

Comment (pbchase) [reply adhoc_check_for_missing__cleared_for_campus_timestamp__when__cleared_for_campus__is_set.R:52]: Addressed in a052941

**Comment by pbchase** [comment]
@ljwoodley I addressed your feedback. Thanks for that. :-)

**Pull request merged**